ledger/participant-integration-api/src/main/scala/platform/store/backend/common/MeteringStorageBackendTemplate.scala

```diff
@@ -200,6 +201,20 @@ private[backend] object MeteringStorageBackendWriteTemplate extends MeteringStor
       .asVectorOf(transactionMeteringParser)(connection)
   }
 
+  def deleteTransactionMetering(from: Offset, to: Offset)(
+      connection: Connection
+  ): Unit = {
```

Comment:
unnecessary curlies

Reply:
True but I always use curlies for functions unless they are one-liners.